src/components/SingleReview.jsx

```diff
@@ -0,0 +1,59 @@
+import { useParams } from 'react-router-dom';
+import { getReviewById } from '../utils';
+import { useState, useEffect } from 'react';
+import CategoryBar from './CategoryBar';
+
+const SingleReview = ({
+  isLoading,
+  setIsLoading,
+  categories,
+  setCategories,
+}) => {
+  const [review, setReview] = useState([]);
+
+  const { review_id } = useParams();
+
+  useEffect(() => {
+    setIsLoading(true);
+    getReviewById(review_id).then((reviewFromApi) => {
+      setReview(reviewFromApi[0]);
+      setIsLoading(false);
+    });
+  }, [review_id]);
+
+  console.log(review);
+  const {
+    title,
+    review_img_url,
+    designer,
+    category,
+    owner,
+    votes,
+    comment_count,
+    review_body,
+  } = review;
+
+  if (isLoading) {
```

Comment:
Loving the isLoading implementation here to provide visual feedback for the user 😄